includes/admin/plugins.php

```diff
@@ -325,6 +324,12 @@ function perflab_render_plugin_card( array $plugin_data ): void {
 			'<button type="button" class="button button-disabled" disabled="disabled">%s</button>',
 			esc_html( _x( 'Active', 'plugin', 'default' ) )
 		);
+		$plugin_path    = $plugin_data['slug'] . '/load.php';
```

Comment:
Note that not every plugin uses `load.php` as the plugin file. Better to not hard-code this.

Reply:
I agree, what would be the best way to not hard code this?

Reply:
You can use `array_keys( get_plugins() )` to get a list of all the plugin files, and then you can get the dirname of each to find the slugs to come up with a mapping of slug to file (e.g. `auto-sizes` to `auto-sizes/auto-sizes.php`)

Reply:
Made the changes by fetching the path from the installed plugins.